extendr-api/tests/dataframe_tests.rs

```diff
@@ -51,3 +51,39 @@ fn test_into_robj_dataframe() {
 
     }
 }
+
+#[test]
+fn test_storing_external_ptr_as_row() {
+    struct Row {
+        id: u32,
+    }
+
+    #[extendr]
+    impl Row {}
+
+    #[extendr]
+    fn use_dataframe_exptr(data_frame: Dataframe<Row>) -> Dataframe<Row> {
+        data_frame
+    }
+}
+
+#[test]
+fn test_dataframe_row_and_externalptr() {
```

Comment:
These tests do not really test anything other than code generation. I'd make them 'integration' tests.

Reply:
They are already in integration tests, as they are stored in `extendr-api/tests` folder.

Reply:
I meant on R side. Here these functions just 'compile', there are no real 'tests' her IMO.

Reply:
They didn't used to compile. They compile now. I'm not going to add more code here.